### ISSUE TITLE
Visual cues on settings/feature-flag page

### ISSUE THREAD

**Issue by efahlstrom** [backstage/backstage#6315]
## Feature Suggestion

When I have the settings/feature-flag page open in full screen it is hard to know which toggle will enable/disable which feature due to many feature-flags and lack of visual cues. 

One possible solution would be to display which feature will be enabled/disabled when hovering above each toggle. Other options would be to group the flags, put the toggles closer to the feature flag names or otherwise make it clearer to see what toggle belong to what flag.

**Comment by freben** [comment]
Yep I've been feeling that maybe the toggles could go to the left of the text actually. Highlight-on-hover for each row works too, maybe not very touchscreen friendly but that's not a big use case for us

**Comment by OrkoHunter** [comment]
Looks like this was implemented in https://github.com/backstage/backstage/pull/6570
